canonical-web-and-design/ubuntu.com
# Replace discourse with launchpad answers for Helping hands

## Done

- Replace discourse with launchpad answers for Helping hands as it was attracting technical support requests that the would never be answered on discourse

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/desktop
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that discourse was replaced in the Helping Hands section


## Issue / Card

Fixes #9339

## Screenshots

![image](https://user-images.githubusercontent.com/441217/110920847-c524b080-8315-11eb-80fe-223560362af2.png)
